src/plugins/owner/SingleOwnerPlugin.sol

```diff
@@ -239,6 +239,23 @@ contract SingleOwnerPlugin is BasePlugin, ISingleOwnerPlugin, IERC1271 {
         return manifest;
     }
 
+    /// @inheritdoc BasePlugin
+    function onReplaceForOldPlugin()external override {
+        _owners[msg.sender] = address(0);
+    }
+
+    /// @inheritdoc BasePlugin
+    function onReplaceForNewPlugin(bytes calldata data) external override {
+        address newOwner = abi.decode(data, (address));
+        _owners[msg.sender] = newOwner;
+    }
+
+    /// @inheritdoc BasePlugin
+    function getDataForMigration() external view override returns (bytes memory){
```

Comment:
I think `getDataForReplacement` may be more appropriate, how do you think?

Reply:
Agree with that. Will edit before merge.